pyproject.toml

```diff
@@ -23,11 +23,13 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
-Flask = "^2.2.2"
-werkzeug = "2.3.7"
-Flask-SocketIO = "^5.3.2"
-flask-sqlalchemy = "^3.0.2"
-Flask-Security-Too = "^5.0.2"
+Flask = "^3.0.3"
+werkzeug = "^3.0.3"
+requests = "^2.32.2"
```

Comment:
We do not import `requests` explicitly. It would be good to have a comment explaining why we list it as direct dependency.

Reply:
Ill add a comment on that, its a security update seems like the libraries using `requests` have not updated to the latest version (https://github.com/mxcube/mxcubeweb/pull/1238)

Reply:
Actually it turns out that is fine now, both `Poetry` and `Sphinx` that uses `requests` should install a more recent version of `requests`